<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>11</y>
      <width>581</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/pngwing.com (7).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>518</x>
      <y>170</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>212</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>213</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>264</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_up">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>202</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_left">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_right">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_down">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_optins">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>224</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_optins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>226</y>
      <width>75</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>btn_start</string>
    </property>
   </widget>
   <widget class="QLabel" name="stick_L">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>49</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pngwing.com (5) (0_0).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stick_r">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pngwing.com (5) (0_0).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="conn_status">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>821</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
